database_cleanup/models/create_indexes.py

```diff
@@ -29,11 +29,9 @@ def purge(self):
                 ),
             )
             tables.add(model._table)
-        for table in tables:
+        for _table in tables:
```

Comment:
Guessing you added the underscore because the variable is not changed? I'm thinking that is a bug and the table variable should be passed instead of model._table in the line below. Can you confirm @hbrunn?

Reply:
> Guessing you added the underscore because the variable is not changed? I'm thinking that is a bug and the table variable should be passed instead of model._table in the line below. Can you confirm @hbrunn?
> As always, you can blame it on the reviewers (#736)

Well to tell the truth I do not remember. But I guess the reason was the following line which reads:
`self.env.cr.execute("analyze %s", (IdentifierAdapter(model._table),))`
So without that change the table variable was never used.

Reply:
Indeed, but the point is that `_table` is still not used, but `model._table` is used instead.

Reply:
Oops, you're right. Now that's strange as original code did the same AFAICT. So this probably needs more investigation to get it right.

Reply:
Yes, it was there from the original PR on. But logically, it only makes sense to use `table` there instead of `model._table`.

Reply:
> Yes, it was there from the original PR on. But logically, it only makes sense to use `table` there instead of `model._table`.

Sure, will do.